assets/pages/paper-subscription-landing/components/tabs.jsx

```diff
@@ -2,32 +2,71 @@
 
 // ----- Imports ----- //
 
-import React, { Component } from 'react';
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 
 import { outsetClassName } from 'components/productPage/productPageContentBlock/productPageContentBlock';
 import ProductPageTabs from 'components/productPage/productPageTabs/productPageTabs';
+import { type PaperDeliveryMethod } from 'helpers/subscriptions';
+import { paperSubsUrl } from 'helpers/routes';
 
-class Tabs extends Component<any, any> {
-  state = {
-    active: 0,
-  }
-  onChange(active: any) {
-    this.setState({ active });
-  }
-  render() {
-    return (
-      <div className={outsetClassName}>
-        <ProductPageTabs
-          active={this.state.active}
-          onChange={(t) => { this.onChange(t); }}
-          tabs={[
-            { name: 'Voucher' },
-            { name: 'Home Delivery' },
-          ]}
-        />
-      </div>
-    );
-  }
-}
-
-export default Tabs;
+import { type State } from '../paperSubscriptionLandingPageReducer';
+import { setTab, type TabActions } from '../paperSubscriptionLandingPageActions';
+
+// ----- Tabs ----- //
+
+export const tabs: {[PaperDeliveryMethod]: {name: string, href: string}} = {
+  collection: {
+    name: 'Voucher',
+    href: paperSubsUrl('collection'),
+  },
+  delivery: {
+    name: 'Home Delivery',
+    href: paperSubsUrl('delivery'),
+  },
+};
+
+type StatePropTypes = {|
+  selectedTab: number,
+|};
+
+type DispatchPropTypes = {|
+  setTabAction: (PaperDeliveryMethod) => *,
+|};
+
+type PropTypes = {|
+  ...StatePropTypes,
+  ...DispatchPropTypes,
+|};
+
+// ----- Component ----- //
+
+const Tabs = ({ selectedTab, setTabAction }: PropTypes) => (
+  <div className={outsetClassName}>
```

Comment:
Can we avoid using component-related classes outside of the components?

Reply:
the idea of this classname is exactly to interplay with the component that exports it (it cancels out the padding, preventing hardcoding it in the sub component) it can become a component however (a div wrapping any children) which is what I've been doing with more involved components like `productPageContentBlockDivider`